AuthenticatorShared/UI/Vault/ItemList/ItemList/ItemListProcessor.swift

```diff
@@ -259,6 +267,19 @@ final class ItemListProcessor: StateProcessor<ItemListState, ItemListAction, Ite
         return []
     }
 
+    /// Subscribe to receive foreground notifications so that we can refresh the item list when the app is relaunched.
+    ///
+    private func setupForegroundNotification() {
+        services.notificationCenterService
+            .willEnterForegroundPublisher()
+            .sink { [weak self] in
+                Task {
+                    await self?.perform(.refresh)
+                }
+            }
```

Comment:
⛏️ I think we should just break sooner if `self` is nil with a guard so it doesn't create the `Task` in that case.
```suggestion
            .sink { [weak self] in
                guard let self else { return }
                Task {
                    await self.perform(.refresh)
                }
            }
```